schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/stac-to-iso19115-3.xsl

```diff
@@ -0,0 +1,322 @@
+                xmlns:lan="http://standards.iso.org/iso/19115/-3/lan/1.0"
+                xmlns:mcc="http://standards.iso.org/iso/19115/-3/mcc/1.0"
+                xmlns:mco="http://standards.iso.org/iso/19115/-3/mco/1.0"
+                xmlns:mdb="http://standards.iso.org/iso/19115/-3/mdb/2.0"
+                xmlns:mrd="http://standards.iso.org/iso/19115/-3/mrd/1.0"
+                xmlns:mri="http://standards.iso.org/iso/19115/-3/mri/1.0"
+                xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
+                xmlns:gex="http://standards.iso.org/iso/19115/-3/gex/1.0"
+                xmlns:gml="http://www.opengis.net/gml/3.2"
+                xmlns:java-xsl-util="java:org.fao.geonet.util.XslUtil"
+                exclude-result-prefixes="#all">
+
+  <xsl:import href="ISO19139/utility/create19115-3Namespaces.xsl"/>
+  <xsl:import href="../process/add-stac-contact.xsl"/>
+  <xsl:import href="../process/add-stac-provider.xsl"/>
+  <xsl:output method="xml" indent="yes"/>
+  <xsl:strip-space elements="*"/>
+
+  <xsl:template match="/">
+    <xsl:choose>
+      <xsl:when test="/record">
+        <xsl:apply-templates select="/record"/>
+      </xsl:when>
+    </xsl:choose>
+  </xsl:template>
+
+  <xsl:template match="/record|*">
+    <mdb:MD_Metadata>
+      <xsl:call-template name="add-iso19115-3.2018-namespaces"/>
+      <mdb:metadataIdentifier>
+        <mcc:MD_Identifier>
+          <mcc:code>
+            <gco:CharacterString>
+              <xsl:value-of select="if ((id)[1] and string-length((id)[1]) > 0) then (id)[1] else generate-id()"/>
+            </gco:CharacterString>
+          </mcc:code>
+        </mcc:MD_Identifier>
+      </mdb:metadataIdentifier>
+      <mdb:defaultLocale>
+        <lan:PT_Locale>
+          <lan:language>
+            <lan:LanguageCode codeList="codeListLocation#LanguageCode"
+                              codeListValue="{java-xsl-util:threeCharLangCode(
+                                (language)[1])}"/>
+          </lan:language>
+          <mdb:hierarchyLevel>
+            <mdb:MD_ScopeCode codeListValue="series" codeList="http://www.isotc211.org/2005/resources/codeList.xml#MD_ScopeCode"/>
+          </mdb:hierarchyLevel>
+          <mdb:hierarchyLevelName>
+            <gco:CharacterString>Data collection</gco:CharacterString>
+          </mdb:hierarchyLevelName>
+          <lan:characterEncoding>
+            <lan:MD_CharacterSetCode codeList="codeListLocation#MD_CharacterSetCode"
+                                     codeListValue="utf8"/>
+          </lan:characterEncoding>
+        </lan:PT_Locale>
+      </mdb:defaultLocale>
+
+      <mdb:metadataScope>
+        <mdb:MD_MetadataScope>
+          <mdb:resourceScope>
+            <mcc:MD_ScopeCode
+              codeList="http://standards.iso.org/iso/19115/resources/Codelists/cat/codelists.xml#MD_ScopeCode"
+              codeListValue="series"/>
+          </mdb:resourceScope>
+        </mdb:MD_MetadataScope>
```

Comment:
A `mdb:name` element is required alongside a `mdb:resourceScope` one for it be valid with regards to ISO rules.

Something like:
```xml
      <mdb:name>
        <gco:CharacterString>Data collection</gco:CharacterString>
      </mdb:name>
```